### PR TITLE
Ensure timeline chat

### PR DESCRIPTION
**Pull request by cammellos**
If the user is asking for timeline data, we ensure it exists before pulling messages.

### CODE REVIEW

File: protocol/messenger.go [hunk 1]
```diff
@@ -3055,7 +3055,16 @@ func (m *Messenger) MessageByChatID(chatID, cursor string, limit int) ([]*common
 		return nil, "", err
```
Comment (flexsurfer) [inline protocol/messenger.go:3055]: won't it return error here if no chat ?

**Review by flexsurfer — APPROVED** [review]

File: protocol/chat.go [hunk 2]
```diff
@@ -33,7 +33,10 @@ const (
 	ChatTypeCommunityChat
 )
 
-const pkStringLength = 68
+const (
+	timelineChatID = "@timeline70bd746ddcc12beb96b2c9d572d0784ab137ffc774f5383e50585a932080b57cca0484b259e61cecbaa33a4c98a300a"
```
Comment (jakubgs) [inline protocol/chat.go:38]: What is this magical string?

File: protocol/messenger.go (hunk 1 above)
Comment (cammellos) [reply protocol/messenger.go:3055]: no, returns nil

File: protocol/chat.go (hunk 2 above)
Comment (jakubgs) [reply protocol/chat.go:38]: A comment would be nice explaining the purpose.

Comment (cammellos) [reply protocol/chat.go:38]: done, is a magical string basically, can be anything, as long as it's a const

**Review by jakubgs — APPROVED** [review]

**Comment by flexsurfer** [comment]
replaced by https://github.com/status-im/status-react/pull/12037

**Pull request closed without merging**